gratipay/utils/fake_data.py

```diff
@@ -309,9 +309,14 @@ def populate_db(db, num_participants=100, ntips=200, num_teams=5, num_transfers=
 
     print("Making Teams")
     teams = []
+    num_teams -= 1      # Make room for the fake Grtipay Team 
```

Comment:
@rohitpaulk I did that because of the test for the fake data. It checked to see if `num_teams` created were correct. Since I had to created the Gratipay Teams separate from the random generated Teams and Owner I made room so that the `fake_data` tests would still pass without the need to change the test. 
I am open to suggestions as to how to do this more elegantly. I now that it is not the ideal work around. 
Another way I was thinking would look something like this  
```
    print("Making Teams")
    teams = []
    teamowners = random.sample(participants, num_teams)
    for teamowner in teamowners:
        if num_teams > 1:
            teams.append(fake_team(db, teamowner))
        else:
            # Creating a fake Gratipay Team
            teams.append(fake_team(db, teamowner, "Gratipay"))
        num_teams -= 1
```